samples/table/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 kotlin {
-    jvm()
+    jvm { withJava() }
```

Comment:
This doesn't seem to belong here. It was anyway merged already as part of https://github.com/ajalt/mordant/pull/124.